### DISCUSSION TITLE
Display Managed Metadata Field in Search Result

### DISCUSSION THREAD

**Discussion by santoshk10** [microsoft-search/pnp-modern-search#3852] [Q&A]
Hi All,

I want to show the Managed Metadata column/Taxonomy field in the search results, which may be multiple or single. I tried using Adaptive Cards but couldn't get it to work. Can you please suggest how to display the Managed Metadata column in the search result web part below the title column?

Thank you!

**Comment by kasperbolarsen** [comment]
You might find some inspiration in the layout samples repository,  perhaps this one,https://github.com/microsoft-search/pnp-modern-search-layouts/blob/main/Results%2FHandlebars%2FList%20Tag%20options%2FListTagOptions.md

It is using handlebars, not the Adaptive Cards, as I find the Adaptive Cards design tooling and experience to lacking

**Reply by santoshk10** [reply]
Hi i tried above but it was showing all managed metadata columns in single line. I have multiple columns and need to show each on separate line. 

 I tried changing data source for Tags it was working when i add new data source it was not working
![image](https://github.com/microsoft-search/pnp-modern-search/assets/29335501/002cf243-b518-497f-84b4-2327801b4c58)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/29335501/afde59c2-a0e1-4c27-8293-fe7ddcce75d2)

It showing only Tags data result 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/29335501/6e0ab78b-fe41-4e9a-8e23-ef3015615ddb)
